Assets/Scenes/Pilot/PilotManager.cs

```diff
@@ -60,4 +60,9 @@ private void EndGame(bool gameEndedSuccessfully)
             SceneManager.LoadScene("GameOver");
         }
     }
+
+     private void FixedUpdate()
+    {
+        DepleteFuel(0.00007f);
```

Comment:
We should probably extract this number into a variable and then multiply it by `Time.FixedDeltaTime` here. Could you change that?